---
title: "Class13"
author: "Matt DuPrie"
date: "May 16, 2018"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# 1000 Genome project data

Read 1000 genome data for MXL dataset
```{r}
genotype <- read.csv("373531-SampleGenotypes-Homo_sapiens_Variation_Sample_rs8067378.csv")
```

```{r}
table(genotype[,2])/ nrow(genotype)*100
```

## Base quality scores from fastqsanger

```{r}
install.packages("seqinr")
install.packages("gtools")

```



```{r}
library(seqinr) 
library(gtools) 
phred <- asc( s2c("DDDDCDEDCDDDDBBDDDCC@") ) - 33 
phred
```


## RNA-seq results
Assessing genetic differences on a population scale.

```{r}
geno <- read.table("rs8067378_ENSG00000172057.6.txt")


```

```{r}
x <- summary( geno$exp[geno$geno == "A/A"])
y <- summary( geno$exp[geno$geno == "A/G"])
summary( geno$exp[geno$geno == "G/G"])
```


Let's make a boxplot

```{r}

boxplot(exp ~ geno, data = geno, notch = TRUE)


```

# Install ggplot2
```{r}
install.packages("ggplot2")
```

```{r}
library(ggplot2)
```


```{r}
ggplot(geno, aes(geno, exp)) + geom_boxplot()
```


```{r}
ggplot(geno, aes(exp, fill = geno)) +geom_density(alpha = 0.2)
```



